src/main/java/top/jsoft/homework/task10/phonebook/PhoneDirectory.java

```diff
@@ -0,0 +1,37 @@
+package top.jsoft.homework.task10.phonebook;
+
+import lombok.Data;
+import lombok.ToString;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by psygrammator
+ * group jsoft.top
+ */
+@Data
+@ToString
+public class PhoneDirectory {
+    private List<Record> records = new ArrayList<>();
+    public void add(Record record)
+    {
+        if(records.stream().filter(name -> name.getName().equalsIgnoreCase(record.getName())).toList().size() < 2)
```

Comment:
What if size > 2? You will not add the new record? Why? You can add records with same names but different phone numbers

Reply:
my bad :/
fixed